flight/views.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+from django.http        import JsonResponse
+from django.views       import View
+
+from .models      import Airline, Airport, FlightPrice, FlightSchedule, FlightStatus
+
+class FlightView(View):
+    def get(self, request):
+        try:
+            data        = request.GET
+            arrival     = data.get('arrival')
+            date        = data.get('date')
+            departure   = data.get('departure')
+            sort        = data.get('sort')
+            time_option = data.getlist('timeOption[]')
```

Comment:
```suggestion
            time_option = data.getlist('timeOption[]')
```
네이밍이 이렇게 된 이유를 알 수 있을까요? 
리스트로 들어가는건 그냥 timeOption에 여러번 전달해주면 리스트로 들어가는데 []로 모양 표시를 하는 이유가 궁금합니다.

Reply:
전달하는 방법을 여러가지로 테스트해보다가 그냥 여러번 전달했을 때 실패하고 []를 붙였을 때 성공해서 []를 붙여 네이밍을 했습니다. 다시 점검하겠습니다. 감사합니다!

Reply:
당시 잘못 테스트했던 것 같습니다. []를 제거하였습니다. 감사합니다!